sway/input/seatop_resize_tiling.c

```diff
@@ -7,6 +7,7 @@
 #include "sway/tree/arrange.h"
 #include "sway/tree/container.h"
 #include "sway/tree/view.h"
+#include "sway/desktop/transaction.h"
```

Comment:
Sorry, one more reorder.

Reply:
To be fair I should've looked for more myself. :/